ci-operator/config/eclipse-che/che-operator/eclipse-che-che-operator-master__v7.yaml

```diff
@@ -58,6 +58,37 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: optional-operators-ci-aws
+- as: devworkspace-happy-path
+  cron: 0 */24 * * *
+  steps:
+    cluster_profile: aws
+    env:
```

Comment:
We do not need to install catalog source because you are using chectl for deploy

Reply:
Example: https://github.com/openshift/release/blob/master/ci-operator/config/eclipse-che/che-operator/eclipse-che-che-operator-7__v8.yaml#L24-L41